03-task-manager/starter/controllers/tasks.js

```diff
@@ -8,12 +8,21 @@ const getAllTasks = asyncWrapper (async (req, res) => {
     res.status(200).json({tasks})  // ({ tasks : tasks}) ES6
     //res.status(200).json({tasks , amount:tasks.length}) 
     //res.status(200).json({success: true , data:{tasks , nbHits:tasks.length}}) 
+  
+    console.log(req.ip)
 })
 
 // What was the reason he used the async and awite here???
 const createTask = asyncWrapper (async ( req, res) => {
-        const task = await Task.create(req.body)
+    const regExp = /[A-Z]/ ;
+    const task = await Task.create(req.body)
+    const isMatch = regExp.test(task.name.charAt(0))
+    if(!isMatch){
```

Comment:
Awesome job!! I appreciate you taking on the optional assignment!!

Reply:
Also this solution looks great!

Reply:
Thanks Jack , I really appreciate your reviews , I didn't had any assignment reviewer before. So I thought there are no comments on my assignments , I am really sorry but I noticed all of this comments accidently tonight!!
much appreciate it!